infrasim/__init__.py

```diff
@@ -35,6 +35,28 @@ def run_command(cmd="", shell=True, stdout=subprocess.PIPE, stderr=subprocess.PI
     return 0, cmd_result[0]
 
 
+def run_command_anyway(cmd="", shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE):
```

Comment:
exception can be ignored in your code, is it necessary to add a new function for this?